### PR TITLE
[ar]: Translate new keys

### PR DESCRIPTION
**Pull request by mohamedsabil83**

### CODE REVIEW

File: locales/ar/json.json [hunk 1]
```diff
@@ -906,8 +906,8 @@
     "The resource was attached!": "تم إرفاق المصدر!",
     "The resource was prevented from being saved!": "تم منع عملية حفظ المصدر!",
     "The resource was updated!": "تم تحديث المصدر!",
-    "The response is not a streamed response.": "The response is not a streamed response.",
-    "The response is not a view.": "The response is not a view.",
+    "The response is not a streamed response.": "الاستجابة ليست استجابة متدفقة",
```
Comment (andrey-helldar) [inline locales/ar/json.json:909]: Is a full stop at the end of the sentence needed here or is it not necessary?

File: locales/ar/json.json [hunk 2]
```diff
@@ -906,8 +906,8 @@
...
+    "The response is not a view.": "الاستجابة ليست صفحة عرض",
```
Comment (andrey-helldar) [inline locales/ar/json.json:910]: Is a full stop at the end of the sentence needed here or is it not necessary?

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/ar/json.json (hunk 1 above)
Comment (mohamedsabil83) [reply locales/ar/json.json:909]: It's not necessary but I forgot it. I'll add it to match the original.

File: locales/ar/json.json (hunk 2 above)
Comment (mohamedsabil83) [reply locales/ar/json.json:910]: It's not necessary but I forgot it. I'll add it to match the original.

**Review by andrey-helldar — APPROVED** [review]
شكراً

**Comment by mohamedsabil83** [comment]
Пожалуйста

**Pull request merged**